docs/system_overview/configuration.md

```diff
@@ -1,6 +1,6 @@
 #
 
-The following configuration files control nearly all aspects of how an iRODS deployment functions. All JSON files validate against the  [configuration schemas in each installation](https://github.com/irods/irods/tree/main/schemas/configuration/v4).
+The following configuration files control nearly all aspects of how an iRODS deployment functions. All JSON files validate against the  [configuration schemas in each installation](https://github.com/irods/irods/tree/main/schemas/configuration/v5).
```

Comment:
`main` -> `5.0.0`

Reply:
one reason to leave it as main might be that we tweak something after release, and then we don't have to remember to come back here and touch the link.

Reply:
True, but that also has the downside of making admins aware of properties their version doesn't support. They may be running iRODS 5, but looking at a schema for 6.0 without knowing it.

Reply:
oh.. so then we have a stable branch...   i was just hesitating on the very specific tag link that you had suggested.

Reply:
Leaving as is. We can adjust the links once 6.0 becomes a thing.